src/styles/variables.scss

```
@@ -0,0 +1,5 @@
+$color-blue-accent: #00acdc;
+$color-main-accent: #060b35;
+$color-gray: #616070;
+$transition-duration: 0.3s;
+$card-width: 200px;
```

It is better to use a more universal naming so that the property of this variable can be changed at any time without changing the name.

Example:

```suggestion
$color-second-accent: #00acdc;
$color-main-accent: #060b35;
$color-title: #616070;
$transition-duration: 0.3s;
$card-width: 200px;
```